.editorconfig

```diff
@@ -2,6 +2,7 @@ root = true
 
 [*]
 indent_style = space
+indent_size = 2
```

Comment:
Unfortunately, this is messing up the consistency with the existing code base in intellij. E.g. wrapped method signatures currently are indented with 4 spaces.

Reply:
Ok, but what’s the point on using editorconfig when the editor should not use it.

I'm using IntelliJ and with the given config, all indentation is set to 4 b default.

In one of the more recent versions Intellj, more detailed settings for editorconfig is supported (https://www.jetbrains.com/help/idea/configuring-code-style.html).

I can revert it, but how can I format it automatically then?

Reply:
Hi Jan,
sorry that this review is taking a little longer - busy days.
I will propose adding intelliJ specific entries to the editor config either later tonight or tomorrow.

Reply:
No problem, I really appreciate all your feedback. You're all doing a great job here, working on the weekends and late each day.

Reply:
By the way, if you have a recent version of IntelliJ and a working ImtelliJ code style configuration, you can just create a new editorconfig file (New -> editorconfig) and it will
generate all the necessary entries.

Generally speaking, the easiest would be to just go with the language default settings and don’t customize. But I guess that’s not an option...